### PR TITLE
refactor: lint code

### PR DESCRIPTION
**Pull request by Aybrea**
## Checklist

List of tasks you have already done and plan to do.

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

Code changes after running the linting script.

### CODE REVIEW

File: packages/varlet-ui/src/menu-select/docs/en-US.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ const value = ref()
 | `label` | The text that the option displays | _any_ | `-` |
 | `value` | The value of the option binding | _any_ | `-` |
 | `disabled` | Whether to disable | _boolean_ | `false` |
-| `ripple` ***3.3.0***  | Whether to enable ripple | _boolean_ | `true` |
+| `ripple` _**3.3.0**_  | Whether to enable ripple | _boolean_ | `true` |
```
Comment (haoziqaq) [inline packages/varlet-ui/src/menu-select/docs/en-US.md:250]: `_` is unnecessary.

Comment (haoziqaq) [reply packages/varlet-ui/src/menu-select/docs/en-US.md:250]: `***` is specific flag.

**Comment by haoziqaq** [comment]
please resolve conflicting files, thx.

**Comment by Aybrea** [comment]
> please solve conflicting files, thx.

done

**Review by haoziqaq — APPROVED** [review]

**Pull request merged**